Dockerfile.test

```diff
@@ -0,0 +1,12 @@
+FROM registry.redhat.io/ubi9/nodejs-18:latest AS web-builder
+
+WORKDIR /usr/src/app
+
+USER 0
+
+COPY package*.json .
```

Comment:
This also needs to consider the `web` path

Reply:
Good point. I updated this in the latest commit.